korge/src/commonMain/kotlin/com/soywiz/korge/view/tiles/TileMap.kt

```diff
@@ -13,23 +13,22 @@ import com.soywiz.korim.color.*
 import com.soywiz.korma.geom.*
 import kotlin.math.*
 
-inline fun Container.tileMap(map: IntArray2, tileset: TileSet, repeatX: TileMap.Repeat = TileMap.Repeat.NONE, repeatY: TileMap.Repeat = repeatX, callback: @ViewDslMarker TileMap.() -> Unit = {}) =
-	TileMap(map, tileset).repeat(repeatX, repeatY).addTo(this, callback)
+inline fun Container.tileMap(map: IntArray2, tileset: TileSet, repeatX: TileMap.Repeat = TileMap.Repeat.NONE, repeatY: TileMap.Repeat = repeatX, smoothing: Boolean = true, callback: @ViewDslMarker TileMap.() -> Unit = {}) =
+	TileMap(map, tileset, smoothing).repeat(repeatX, repeatY).addTo(this, callback)
 
-inline fun Container.tileMap(map: Bitmap32, tileset: TileSet, repeatX: TileMap.Repeat = TileMap.Repeat.NONE, repeatY: TileMap.Repeat = repeatX, callback: @ViewDslMarker TileMap.() -> Unit = {}) =
-	TileMap(map.toIntArray2(), tileset).repeat(repeatX, repeatY).addTo(this, callback)
+inline fun Container.tileMap(map: Bitmap32, tileset: TileSet, repeatX: TileMap.Repeat = TileMap.Repeat.NONE, repeatY: TileMap.Repeat = repeatX, smoothing: Boolean = true, callback: @ViewDslMarker TileMap.() -> Unit = {}) =
+	TileMap(map.toIntArray2(), tileset, smoothing).repeat(repeatX, repeatY).addTo(this, callback)
 
 @PublishedApi
 internal fun Bitmap32.toIntArray2() = IntArray2(width, height, data.ints)
 
 @OptIn(KorgeInternal::class)
-open class TileMap(val intMap: IntArray2, val tileset: TileSet) : View() {
+open class TileMap(val intMap: IntArray2, val tileset: TileSet, val smoothing: Boolean) : View() {
     private var contentVersion = 0
-	constructor(map: Bitmap32, tileset: TileSet) : this(map.toIntArray2(), tileset)
+	constructor(map: Bitmap32, tileset: TileSet, smoothing: Boolean) : this(map.toIntArray2(), tileset, smoothing)
```

Comment:
To be backwards compatible, can you add default values to this constructors too?
The existing property was mutable, can you mark it as `var` instead of `val`?